### ISSUE TITLE
Inserting images 

### ISSUE THREAD

**Issue by lenisko** [trikita/slide#6]
It is possible to add this future?

**Comment by zserge** [comment]
Images are definitely planned.

We are thinking about the following syntax `@imagepath`.
At the beginning of the paragraph this would mean a background image, otherwise it will be an inline image.
"imagepath" is a URI (either web, or local file system). This should allow inserting photos from the gallery or any other apps as well as using funny meme pictures or whatever there can be found around the web.

This means that Slide will get a way to attach a picture to a slide, but will have no builtin image editor etc - this is far beyond its current scope.

**Comment by lenisko** [comment]
Glad to hear it!

I think that is good idea, but maybe some size limiting like @imagepath:/home/...;30% which would resize image to 30% of slide space would be good to?
But right that might kill intent which is very simple application for slides.

**Comment by zserge** [comment]
Fixed in version 1.2. The syntax is as follows:

```
@http://example.com/some_image.png center 30%
Some text on top of the image
@http://example.com/logo.png nw 10%
```

Multiple images per slide are allowed, they will be stacked. Each image may have optional gravity (center by default) and scale (100% by default which is fit-to-screen). Gravity can be specified as left/top/right/bottom or as n/w/s/e/nw/ne/sw/se.

Local images are allowed, they will be also represented by some content provider URL.
